debs/cloud-info-provider-opennebula/debian/changelog

```diff
@@ -1,3 +1,10 @@
+python-cloud-info-provider-opennebula (0.11.1-1) xenial; urgency=medium
```

Comment:
Now that we have the stevedore stuff ready, we should think on moving this to separate packages that can be updated separately.

But, that's low priority at the moment.